### PR TITLE
Fix http test: test-http-same-map.js

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots @cristiancavalli PTAL

File: test/non-interference/http-e2e.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ glob(test_glob, function(err, files) {
           ' >' +  files[testCount] + '.instru.js' + '&& mv ' + files[testCount] +
           '.instru.js' + ' ' + files[testCount]);
     }
-    var results = cp.spawnSync('node', [files[testCount]]);
+    var results = cp.spawnSync('node', ['--allow_natives_syntax', files[testCount]]);
```
Comment (cristiancavalli) [inline test/non-interference/http-e2e.js]: @matthewloring Should the commit comments mention anything about this flag and what it does? I can't find any information about it just from a preliminary google search

Comment (matthewloring) [inline test/non-interference/http-e2e.js]: Updated.

**Comment by cristiancavalli** [comment]
👍  once CI's pass

**Pull request merged**